02-express-tutorial/12-route-app.js

```diff
@@ -0,0 +1,24 @@
+// As we have more routes and more functionalities, our entry point(app.js) gets busy. 
+// So we have to use ExpressJS router to group those routes and set them up as separate
+// controllers in terms of functionalies.
+
+const express = require("express");
+const app = express();
+
+const people = require("./routes/people");
+const auth = require("./routes/auth");
+// static assets
+app.use(express.static("./methods-public"));
+// parse form data
+app.use(express.urlencoded({ extended: false }));
+// parse.json
+app.use(express.json());
+
+// To use app.use() method to set up the base urls
+// [("/api/people") & (/login)] for the people and auth file:
+app.use("/api/people", people); // So you remove "/api/people" from the url in people.js
+app.use("/login", auth);        // So you remove "login" from the url in auth.js
```

Comment:
The answer to your question from last week about handling lots of functionality in a single route! 🎉

Reply:
I remembered asking and I was happy learning this a lesson after.

Thank you Taylor for your timely response and comments.

I appreciate.